.github/workflows/ci.yaml

```diff
@@ -0,0 +1,69 @@
+name: CI
+
+on:
+    push:
+        branches:
+            - main
+            - '[0-9]+.[0-9]+'
+    pull_request: ~
+
+jobs:
+    cs-fix:
+        name: Run code style check
+        runs-on: "ubuntu-20.04"
+        strategy:
+            matrix:
+                php:
+                    - '7.4'
+        steps:
+            - uses: actions/checkout@v2
+
+            - name: Setup PHP Action
+              uses: shivammathur/setup-php@v2
+              with:
+                  php-version: ${{ matrix.php }}
+                  coverage: none
+                  extensions: 'pdo_sqlite, gd'
+                  tools: cs2pr
+
+            - uses: "ramsey/composer-install@v1"
+              with:
+                  dependency-versions: "highest"
+
+            - name: Run code style check
+              run: composer run-script check-cs -- --format=checkstyle | cs2pr
+
+    tests:
+        name: Unit tests
+        runs-on: "ubuntu-20.04"
+        timeout-minutes: 15
+
+        strategy:
+            fail-fast: false
+            matrix:
+                php:
+                    - '5.6'
```

Comment:
Why 5.6?

Reply:
It was this way on Travis: https://github.com/ezsystems/ezplatform-design-engine/blob/2.0/.travis.yml#L9
and I didn't notice that it's unsupported by v2.5

I will remove it, thanks for noticing

Reply:
Removed in https://github.com/ezsystems/ezplatform-design-engine/pull/44/commits/acd6975c46a9f7831ef1f997b50a16ec572fabec